### DISCUSSION TITLE
Add new option for toggling leaderboard elements on/off during break times

### DISCUSSION THREAD

**Discussion by fiery-rage** [ppy/osu#14194] [Ideas]
The [recent update](https://osu.ppy.sh/home/changelog/stable40/20210809.1) to the stable client (specifically moving the key overlay above the flashlight layer) has broken a workaround fix that tournament players have been using for years now. The workaround essentially disables the leaderboard from ever appearing, including during break times. Tournament players primarily use this as a means to be less distracted during break times, as the crown and leaderboard elements take up quite a bit of space and can be easily distracting. This issue was already reported [here](https://github.com/ppy/osu-stable-issues/issues/807).

Unlike that report, however, I'd rather the workaround not be "patched" to work properly. It'd be much more effective to just have the intended purpose of the workaround as a toggleable option in the game itself. You can already toggle leaderboard elements (such as the crown and scores of other players in multiplayer) on/off for gameplay; allowing this to be extended to break times would just be an extension of that, and wouldn't impact overall gameplay or experience. This also would effectively negate the need for the workaround.

**Comment by peppy** [comment]
Is this related to lazer? If not you are likely in the wrong place.

If you are intending to report to https://github.com/ppy/osu-stable-issues/issues, please beware that stable is feature locked and this may not be implemented there. In addition, this is already reported at https://github.com/ppy/osu-stable-issues/issues/807.

**Reply by fiery-rage** [reply]
This was primarily intended for stable as a suggestion (apologies for that -- this is my first time using GitHub, everything's kind of confusing layout-wise), but I'd also ideally like to see this feature implemented in lazer as well.

**Reply by peppy** [reply]
I'd recommend trying lazer first, as the whole display elements are different (and leaderboards already have a "compact" mode which intends to stay out of the way).
